drivers/docker/handle.go

```diff
@@ -246,6 +246,8 @@ func (h *taskHandle) run() {
 	if ierr != nil {
 		h.logger.Error("failed to inspect container", "error", ierr)
 	} else if container.State.OOMKilled {
+		h.logger.Error("OOM Killed", "containerID", h.containerID, "containerImage", h.containerImage)
```

Comment:
our convention is that log context keys should always follow "snake case":

```suggestion
		h.logger.Error("OOM Killed", "container_id", h.containerID, "containerImage", h.containerImage)
```

Reply:
Of course, I've corrected it!

Also added two additional points of data. It's possible to run the same container with different arguments, where some will trigger OOM and other will not. It would be useful to glean as much info as possible from the log.